lib/db_connection.ex

```diff
@@ -804,19 +806,20 @@ defmodule DBConnection do
   ### Options
 
   See module documentation. The pool and connection module may support other
-  options. All options are passed to `handle_status/2`.
+  options. All options are passed to `c:handle_status/2`.
 
   ### Example
 
       {:ok, conn, _result} = DBConnection.begin(pool)
```

Comment:
there's neither `DBConnection.begin` nor `DBConnection.commit` below in the example, need to update this.

Reply:
It should be `DBConnection.transaction`.

Reply:
Hi there - without `begin`/`commit`, how would one use non-closure based transactions, with the new [DBConnection.html.transaction/3](
https://hexdocs.pm/db_connection/2.0.0-rc.0/DBConnection.html#transaction/3)? This is a follow up to #87, which is closed now. Thank you!

Reply:
It is not possible to have non-closure based transactions. It adds to many complications and possible failure scenarios, so we decided to not support it.